partner_contact_department/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Contact department
+==================
+
+This module extends the functionality of the address book to support
+departments.
+
+Department is a drop-down field in partner forms, and it refers to contact
+department in its own company.
+
+Configuration
+=============
+
+To configure departments, you need to:
+
+* Go to *Sales > Configuration > Address Book > Departments*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to any partner's form.
```

Comment:
It's too obvious to explain this

![2015-12-22_15-43-50](https://cloud.githubusercontent.com/assets/8736623/11957584/e14420ee-a8c2-11e5-8966-37bff1946778.png)